test/conformance/helpers/constants.go

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Since you define them as functions here, do you think `uri.go` is a better file name?

Reply:
Done.